setup.py

```diff
@@ -2,9 +2,19 @@
 # Copyright (C) 2013  Ashton S. Reimer
 # Full license can be found in LICENSE.txt
 import os
-# Need to use the enhanced version of distutils packaged with
-# numpy so that we can compile fortran extensions
 from setuptools import find_packages, setup
+import subprocess
```

Comment:
Yes. Conda is used widely, particularly the 30 MB Miniconda. When you start using Continuous Integration e.g. travis-ci.org also Conda is used there.